server/food_bank_app/users/models.py

```diff
@@ -4,17 +4,18 @@
 from phonenumber_field.modelfields import PhoneNumberField
 # Create your models here.
 
-
 class Contributor(AbstractUser):
-
     id = models.UUIDField(primary_key=True, default=uuid.uuid4)
+    email = models.EmailField(null=False, unique=True,
+                              editable=False, max_length=20)
+
+class ContributorData(models.Model):
```

Comment:
The absence of a ContributorData entity represents a state in which the contributor hasn't set its profile data.

In the ContributorData model, all fields are non-nullable (except for RFC since it's an optional field). A user could later create this record after finishing their signup form, ensuring they send a correct request, since all the fields in this model are non nullable. Another advantage of having a separate Contributor model (only for auth purposes) is that they can perform a signup request before setting their profile data (this could prevent the user to navigate through the signup steps using an invalid email, for example).